src/current/_includes/releases/v24.1/v24.1.3.md

```diff
@@ -0,0 +1,71 @@
+## v24.1.3
+
+Release Date: August 1, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-1-3-sql-language-changes">SQL language changes</h3>
+
+- [`EXPLAIN ANALYZE`]({% link v24.1/explain-analyze.md %}) statements are now supported when executed via UI SQL shell. [#125563][#125563]
+- Added the [`sql.auth.grant_option_inheritance.enabled` cluster setting]({% link v24.1/cluster-settings.md %}#setting-sql-auth-grant-option-inheritance-enabled). The default value is `true`, which maintains consistency with CockroachDB's previous behavior: users granted a privilege with [`WITH GRANT OPTION`]({% link v24.1/grant.md %}) can in turn grant that privilege to others. When `sql.auth.grant_option_inheritance.enabled` is set to `false`, the `GRANT OPTION` is not inherited through role membership, thereby preventing descendant roles from granting the privilege to others. However, the privilege itself continues to be inherited through role membership. [#126300][#126300]
+
+<h3 id="v24-1-3-operational-changes">Operational changes</h3>
+
+- `crdb_internal.cluster_execution_insights.txt` and `crdb_internal.cluster_txn_execution_insights.txt` have been removed from the [debug zip]({% link v24.1/cockroach-debug-zip.md %}). These files contained cluster-wide insights for statements and transactions. Users can still rely on the [per-node execution]({% link v24.1/cockroach-debug-zip.md %}#files) insights in `crdb_internal.node_execution_insights.txt` and `crdb_internal.node_txn_execution_insights.txt`. [#125807][#125807]
+- For the [TELEMETRY channel]({% link v24.1/logging.md %}#telemetry), TCL `sampled_query` events will now be sampled at the rate specified by the setting `sql.telemetry.query_sampling.max_event_frequency`, which is already used to limit the rate of sampling DML statements. [#126729][#126729]
+
+<h3 id="v24-1-3-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where collection of [debug information]({% link v24.1/cockroach-debug-zip.md %}) for very long-running [jobs]({% link v24.1/show-jobs.md %}) could use excessive space in the `job_info` system table and cause some interactions with the jobs system to become slow. [#126122][#126122]
+- Fixed a bug where a change to a [user-defined type]({% link v24.1/create-type.md %}) could cause queries against tables using that type to fail with an error: `histogram.go:694: span must be fully contained in the bucket`. The change to the user-defined type could come directly from an [`ALTER TYPE`]({% link v24.1/alter-type.md %}) statement, or indirectly from an [`ALTER DATABASE ADD REGION`]({% link v24.1/alter-database.md %}#add-region) or [`DROP REGION`]({% link v24.1/alter-database.md %}#drop-region) statement (which implicitly change the [`crdb_internal_region`]({% link v24.1/alter-table.md %}#set-the-table-locality-to-regional-by-row) type). This bug has existed since UDTs were introduced in v20.2. [#125800][#125800]
```

Comment:
suggest adding "user-defined-type (UDT)" so the later mention of "UDT" is defined